scss/main.scss

```diff
@@ -96,6 +96,25 @@ $grey: #C4C4C4;
   @include box($pink);
 }
 
+#navbar {
+  background: #FFFFFF;
+  display: flex;
+  .nav-link {
+    @include dinpro-text(24px,bold,$lavender);
+    text-transform: uppercase;
+  }
+  .nav-link:hover {
+    transition: .3s ease;
+  }
+}
+
+.navbar-nav > li{
+  margin-left: 33px;
+  margin-right: 33px;
+}
+
+.navbar { padding-top: 0.01rem; padding-bottom: 0.01rem; }
```

Comment:
nit: u can just add this in #navbar for cleaner code since #navbar and .navbar are talking abt the same element

Reply:
Oh sorry will do!